api/vudl.ini.dist

```diff
@@ -194,3 +194,118 @@ DOC = "vudl-system:DOCData"
 Audio = "vudl-system:AudioData"
 XLS = "vudl-system:XLSData"
 Text = "vudl-system:TextData"
+
+# AudioData
+[models]
+AudioData.datastreams.WAV.mimetype.allowedType = "audio"
```

Comment:
Any particular reason why WAV is formatted differently than subsequent formats? Should we make a `[models.AudioData.datastreams.WAV.mimetype]` section for consistency with what we're doing below?

Reply:
Sorry. that's a typo.